Fail to create image xrv9k 7.10.1

Tried creating a docker image based on xrv9k-fullk9-x-7.10.1.qcow2 and I'm getting the following below. Could you help me with this ?
```
~/vrnetlab/xrv9k$ make docker-image
for IMAGE in xrv9k-fullk9-x-7.10.1.qcow2; do \
        echo "Making $IMAGE"; \
        make IMAGE=$IMAGE docker-build; \
done
Making xrv9k-fullk9-x-7.10.1.qcow2
make[1]: Entering directory '/home/ismael/vrnetlab/xrv9k'
rm -f docker/*.qcow2* docker/*.tgz* docker/*.vmdk* docker/*.iso
ERROR: Incorrect version string (xrv9k-fullk9-x-7.10.1.qcow2). The regexp for extracting version information is likely incorrect, check the regexp in the Makefile or open an issue at https://github.com/plajjan/vrnetlab/issues/new including the image file name you are using.
make[1]: *** [../makefile.include:25: docker-build-common] Error 1
make[1]: Leaving directory '/home/ismael/vrnetlab/xrv9k'
make: *** [../makefile.include:9: docker-image] Error 2
```

--- a/xrv/Makefile
+++ b/xrv/Makefile
@@ -9,7 +9,7 @@ IMAGE_GLOB=*vmdk*
 # iosxrv-k9-demo-6.2.2.15I.DT_IMAGE.vmdk
 # iosxrv-k9-demo-6.2.2.1T-dhcpfix.vmdk
 # iosxrv-k9-demo-6.2.2.22I.vmdk
-VERSION=$(shell echo $(IMAGE) | sed -e 's/.\+[^0-9][^0-9]\([0-9]\.[0-9]\.[0-9]\(\.[0-9A-Z]\+\)\?\)\([^0-9].*\|$$\)/\1/')
+VERSION=$(shell echo $(IMAGE) | sed -e 's/.\+[^0-9]\([0-9]\+\.[0-9]\+\.[0-9]\+\(\.[0-9A-Z]\+\)\?\)\([^0-9].*\|$$\)/\1/')
 
 -include ../makefile-sanity.include
 -include ../makefile.include

--- a/xrv9k/Makefile
+++ b/xrv9k/Makefile
@@ -7,7 +7,8 @@ IMAGE_GLOB=*qcow2*
 # TODO: add example file names here
 # xrv9k-fullk9-x.vrr-6.1.3.qcow2
 # xrv9k-fullk9-x.vrr-6.2.1.qcow2
-VERSION=$(shell echo $(IMAGE) | sed -e 's/.\+[^0-9]\([0-9]\.[0-9]\.[0-9]\(\.[0-9A-Z]\+\)\?\)\([^0-9].*\|$$\)/\1/')
+# xrv9k-fullk9-x-7.10.1.qcow2
+VERSION=$(shell echo $(IMAGE) | sed -e 's/.\+[^0-9]\([0-9]\+\.[0-9]\+\.[0-9]\+\(\.[0-9A-Z]\+\)\?\)\([^0-9].*\|$$\)/\1/')
 
 -include ../makefile-sanity.include
 -include ../makefile.include

